usr/share/rear/finalize/Fedora/s390/660_install_zipl.sh

```diff
@@ -0,0 +1,67 @@
+#
+# finalize/Linux-s390/660_install_grub2_and_zipl.sh
+#
+# This script is iderived from finalize/Linux-i386/660_install_grub2.sh
+#
+# The generic way how to install the bootloader on s390 on RHEL
+# when one is not "inside" the system
+# but "outside" like in the ReaR recovery system or in a rescue system is
+# to install zipl from within the target system environment via 'chroot'
+# basically via commands like the following:
+#
+#   chroot /mnt/local /sbin/zipl -n
+#
+# where -n mean non-interactive
+# special bootloader setup on s390 on RHEL.
+# On RHEL booting IBM Z basically works this way:
+#   The bootloader (zIPL) is IPL'd which then loads the kernel
+#   the kernel then loads the initramfs and startints running the init scripts
+#   zipl is boot loader
+#   zipl uses /etc/zipl.conf to configure the loader
+#
+# cf. https://github.com/rear/rear/issues/2137#issuecomment-490420041
+# and https://www.ibm.com/support/knowledgecenter/en/linuxonibm/com.ibm.linux.z.lhdd/lhdd_c_ipl_vs_boot.html
+#
+# from man pages:
+#       ┌────────────────┬────────────┬─────────────────────────────────┐
```

Comment:
I think the table lines are made with non-ASCII characters
but there should be not any non-ASCII character in ReaR
even regardless whether or not it is only a comment,
cf. "Character encoding" in
https://github.com/rear/rear/wiki/Coding-Style

So for thingls like lines in a table use only
traditional (7-bit) ASCII charactes like `|` `-` `+` and `_` 
for example like:
```
#       +----------------+------------+---------------------------------+
#       | Arch           | Bootloader | Configuration File              |
#       |----------------+------------+---------------------------------|
#       | x86_64 [BIOS]  | grub2      | /boot/grub2/grub.cfg            |
#       |----------------+------------+---------------------------------|
#       | x86_64 [UEFI]  | grub2      | /boot/efi/EFI/redhat/grub.cfg   |
#       |----------------+------------+---------------------------------|
#       | i386           | grub2      | /boot/grub2/grub.cfg            |
#       |----------------+------------+---------------------------------|
#       | ia64           | elilo      | /boot/efi/EFI/redhat/elilo.conf |
#       |----------------+------------+---------------------------------|
#       | ppc [>=Power8] | grub2      | /boot/grub2/grub.cfg            |
#       |----------------+------------+---------------------------------|
#       | ppc [<=Power7] | yaboot     | /etc/yaboot.conf                |
#       |----------------+------------+---------------------------------|
#       | s390           | zipl       | /etc/zipl.conf                  |
#       |----------------+------------+---------------------------------|
#       | s390x          | zipl       | /etc/zipl.conf                  |
#       +----------------+------------+---------------------------------+
```

```

Reply:
fixed